### PR TITLE
Support running modules

### PR DESCRIPTION
**Pull request by iddan**
Solves #43. It was a pleasure to get into the codebase. Would suggest adding setting up a venv into the readme under "contributing" or something.

### CODE REVIEW

**Comment by iddan** [comment]
@joerick can you please review it?

**Comment by joerick** [comment]
sure thing. thanks for the contribution! i'll get around to it, just busy at work this week. I'll see if i can take a look at the weekend. thanks!

File: pyinstrument/__main__.py [hunk 1]
```diff
@@ -44,64 +45,74 @@ def main():
...
+    sys.argv[:] = args
 
-        if options.output_renderer:
-            renderer = options.output_renderer
-        elif options.output_html:
-            renderer = 'html'
-        else:
-            renderer = 'text'
+    if options.module_name is not None:
+        progname = find_spec(options.module_name).origin
```
Comment (joerick) [inline pyinstrument/__main__.py]: I'm not sure this is right - when I run `find_spec('pyinstrument').origin` in a shell it gives me the `__init__.py` file. It should be running the `__main__.py` file as I understand it.

The [cProfile module](https://github.com/python/cpython/blob/master/Lib/cProfile.py#L157) uses [runpy](https://docs.python.org/2.7/library/runpy.html) for this - it's probably best to copy that.

**Comment by iddan** [comment]
Changed to runpy and made test succeed

**Comment by iddan** [comment]
If you can please post when it is released to PyPI.

**Comment by joerick** [comment]
Sorry for the delay - just addressing a bug I found before releasing... I had to make a minor change to this for programs that use ArgumentParser to work. Most programs assume sys.argv has at least one entry. see 2c0599063ba7ac026aba74249cc0e7ec697f5a6c

**Comment by joerick** [comment]
released as 2.1.1!

**Comment by iddan** [comment]
Maybe update should be state in README?

**Pull request merged**